tests/test_helpers.py

```diff
@@ -0,0 +1,34 @@
+from yatl.helpers import TAG, XML
+import unittest
```

Comment:
excellent tests!

Reply:
I added a new test that checks each tag created with the TAG object, this is only possible if you keep the classmethod _add_tag that adds the tag names created with the TAG object in the __all_tags__ attribute